bootstrap.php

```diff
@@ -0,0 +1,4 @@
+<?php
+
+\Doctrine\Common\Enum\Factory::registerEngine(new \Doctrine\Common\Enum\Engine\SPL());
+\Doctrine\Common\Enum\Factory::registerEngine(new \Doctrine\Common\Enum\Engine\MyCLabs());
```

Comment:
a total -1 for a global state being initialized eagerly for all projects using the library